packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -191,6 +192,12 @@ export default class SchedulerTask {
                               JSON.stringify(sendNowSchedulerFilters),
                           )}`
                         : ''
+                }${
+                    selectedTabs
+                        ? `?selectedTabs=${encodeURI(
+                              JSON.stringify(selectedTabs),
```

Comment:
you don't need to do stringify if you use text[] in db

Reply:
i removed the stringify call in the model layer, but keep this call because it seems a straightforward way to encode the string array and decode it at the endpoint.

Reply:
Ok, that works, another option is to pass an array using the query parameter multiple times like. 

`?selectedTabs=tab0&selectedTabs=tab1...`

Both ways are accepted